src/main/resources/assets/schemas/DarCollection.yaml

```diff
@@ -30,3 +30,11 @@ properties:
     description: List of all datasets referenced across all DARs within the collection
     items:
       $ref: './Dataset.yaml'
+  electionMap:
+    type: object
+    description: Map of referenceIds to their associated elections
+    additionalProperties: 
+      type: array
```

Comment:
I think what you want here is `object` so you can describe a map type. Swagger docs: https://swagger.io/docs/specification/data-models/dictionaries/